Content.Server/GameObjects/Components/Nutrition/DrinkComponent.cs

```diff
@@ -0,0 +1,162 @@
+using System;
+using Content.Server.GameObjects.Components.Sound;
+using Content.Server.GameObjects.EntitySystems;
+using Content.Shared.Chemistry;
+using Content.Shared.GameObjects.Components.Nutrition;
+using Content.Shared.Interfaces;
+using Robust.Server.GameObjects;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Interfaces.GameObjects;
+using Robust.Shared.Serialization;
+using Robust.Shared.ViewVariables;
+
+namespace Content.Server.GameObjects.Components.Nutrition
+{
+    [RegisterComponent]
+    public class DrinkComponent : Component, IAfterAttack, IUse
+    {
+        public override string Name => "Drink";
+        [ViewVariables]
+        public Solution Contents => _contents;
+        private Solution _contents;
```

Comment:
Shouldn't this be a `SolutionComponent` instead? That would make it easier to have chemical reactions occur in food/drink once the chemical reaction system is in. Since presumably the reaction system would iterate over all `SolutionComponent`s and check for reactions each time it's run.

Reply:
I did try that at first but I was getting null reference exceptions on _containedSolution. I'll have another look at using it.